backend/src/libs/packages/server-application/libs/helpers/create-error-response/create-error-response.helper.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  errorHandlers,
+  handleDefaultErrorResponse,
+} from './handlers/handlers.js';
+import { type ErrorParameter, type ErrorResponse } from './types/types.js';
+
+const createErrorResponse = (error: ErrorParameter): ErrorResponse => {
+  const handler =
+    errorHandlers.find((handler) => handler(error)) ??
+    handleDefaultErrorResponse;
```

Comment:
```suggestion
  const handler =
    errorHandlers.find((errorHandler) => errorHandler(error)) ??
    handleDefaultErrorResponse;
```
'handler' is being defined 2 times in your case